src/seedu/addressbook/data/AddressBook.java

```diff
@@ -89,6 +97,14 @@ public void addPerson(Person toAdd) throws DuplicatePersonException {
     public void addTag(Tag toAdd) throws DuplicateTagException {
         allTags.add(toAdd);
     }
+    
+    public void StatusOfTagAdd(Person p, Tag t) {
```

Comment:
the naming of this method is rather funny. usually for a method that does something, we start with a verb.